samples/fpgaperf_counter/fpgaperf_counter.c

```diff
@@ -469,6 +469,15 @@ fpga_result fpgaPerfCounterGet(fpga_token token, fpga_perf_counter *fpga_perf)
 
 	memset(fpga_perf, 0, sizeof(fpga_perf_counter));
 
+	//check if its being run as root or not
```

Comment:
Hi Ramya, rather than checking here in the perf counter library, we should add the privilege check in the host_exerciser app.